src/libs/elektra/key.c

```diff
@@ -455,6 +455,13 @@ Key * keyCopy (Key * dest, const Key * source, elektraCopyFlags flags)
 	return NULL;
 }
 
+/**
+ * Frees the `key`, `ukey` and `data.v` fields from the provided `key` parameter.
+ * `key` and `ukey` are only freed if the flag `KEY_FLAG_MMAP_KEY` is NOT set.
+ * `data.v` is only freed if the flag `KEY_FLAG_MMAP_DATA` is NOT set.
+ * The checks are made separately which makes this call not atomic.
```

Comment:
What do you mean by this? I don't see any return path that would lead to an inconsistent state.